File: src/Phoenix.Models/Plcs/Climatixs/Dto/ClimatixChartDto.cs
using Phoenix.Models.Base.Dto;

namespace Phoenix.Models.Plcs.Climatixs.Dto
{
   public sealed class ClimatixChartDto : PlcChartDtoBase
   {
      public float OutsideTemp { get; init; }
      public float ChHighInletPresure { get; init; }
      public float ChHighOutletPresure { get; init; }

      public float Ch1LowInletTemp { get; init; }
      public float Ch1LowOutletTemp { get; init; }
      public float Ch1LowOutletPresure { get; init; }
      public float Ch1HeatCurveTemp { get; init; }

      public float Ch2LowInletTemp { get; init; }
      public float Ch2LowOutletTemp { get; init; }
      public float Ch2LowOutletPresure { get; init; }
      public float Ch2HeatCurveTemp { get; init; }

      public float DhwTemp { get; init; }
      public float DhwTempSet { get; init; }
   }
}

File: src/Phoenix.Web/src/models/api/plcs/climatixs/dto/climatixChartDto.ts
import { PlcChartDtoBase } from "../../../base/dto/plcChartDtoBase";

export interface ClimatixChartDto extends PlcChartDtoBase {
   outsideTemp: number;
   chHighInletPresure: number;
   chHighOutletPresure: number;
   ch1LowInletTemp: number;
   ch1LowOutletTemp: number;
   ch1LowOutletPresure: number;
   ch1HeatCurveTemp: number;
   ch2LowInletTemp: number;
   ch2LowOutletTemp: number;
   ch2LowOutletPresure: number;
   ch2HeatCurveTemp: number;
   dhwTemp: number;
   dhwTempSet: number;
}
